book/src/Example.md

```diff
@@ -133,6 +133,63 @@ csi-hostpathplugin-9rp7c     2/2     Running   0          5s
 {{#include example/hostpath/csi-hostpathplugin.yaml}}
 ```
 
+
+### Deploy livenessprobe with CSI plugin
+
+Below is an example configuration which needs to be added to CSI driver/provisioner yaml:
+
+Note: This example is derived from [using-livenessprobe](https://github.com/kubernetes-csi/livenessprobe#using-livenessprobe) from [kubernetes-csi/livenessprobe](https://github.com/kubernetes-csi/livenessprobe)
+
+```yaml
+- name: hostpath-driver
+    image: quay.io/k8scsi/hostpathplugin:vx.x.x
+    imagePullPolicy: Always
+    securityContext:
+      privileged: true
+#
+# Defining port which will be used to GET plugin health status
+# 9808 is default, but can be changed.
+#
+    ports:
+    - containerPort: 9808
+      name: healthz
+      protocol: TCP
+    livenessProbe:
+      failureThreshold: 5
+      httpGet:
+        path: /healthz
+        port: healthz
+      initialDelaySeconds: 10
+      timeoutSeconds: 3
+      periodSeconds: 2
+      failureThreshold: 1
+...
+#
+# Spec for liveness probe sidecar container
```

Comment:
I would add some more detail here about what is going on.  This sidecar container is exposing the /healthz endpoint that the livenessProbe above queries.

Reply:
Sure, will add.